src/pytorch_lightning/strategies/ddp_spawn.py

```diff
@@ -315,7 +337,7 @@ def teardown(self) -> None:
             if (
                 _TORCH_GREATER_EQUAL_1_11
                 and not self.model.static_graph
-                and self.model._get_ddp_logging_data().get("can_set_static_graph")
+                and self.model._get_ddp_logging_data().get("can_set_static_graph")  # type: ignore[operator]
```

Comment:
this should work because we have `if isinstance(self.model, DistributedDataParallel)` above. But it might be that our mypy ci runs with torch < 1.11. 
@carmocca any suggestion how to handle it?

Reply:
Mypy expects `model.whatever` to return a tensor (https://github.com/pytorch/pytorch/issues/81462) so ignoring the warning is the correct solution

` error: "Tensor" not callable  [operator]`